docs/explanations/reports.md

```diff
@@ -9,7 +9,7 @@ Curio can generate two types of reports about your codebase, all from the same u
 
 ## Summary Report
```

Comment:
@markmichon some proposed doc updates for your consideration. Let me know if you have any questions or need further context 🙏 
Mainly, I've added a sentence about warning-level violations not triggering the exit failure status, in case we want to include this.

Reply:
@elsapet lgtm. Added the severity flag to the config docs as well and tacked that onto this pr.